config/config.js.sample

```diff
@@ -9,7 +9,11 @@
  */
 
 var config = {
-	address: "localhost",
+	address: "", // Address to listen on, can be:
```

Comment:
I think we need keep the localhost value for config sample... and only add comments here.

Reply:
Okay, sounds fine to me

Reply:
Will you update this in the PR?